mep/people/views.py

```diff
@@ -523,27 +524,29 @@ def get_breadcrumbs(self):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        # use order within manifest to get next/prev links
-        sibling_cards = self.object.manifest.canvases
-        # convert into a list of ids in order to get adjacent ids
-        # that skip over blank cards
-        sibling_cards = list(sibling_cards.values_list('short_id', flat=True))
-        current_index = sibling_cards.index(self.object.short_id)
-        try:
-            next_card = sibling_cards[current_index + 1]
-        except IndexError:
-            next_card = None
-        if current_index > 0:
-            prev_card = sibling_cards[current_index - 1]
-        else:
-            prev_card = None
 
+        # get all cards (canvases) from the manifest and store their ids
+        cards = self.object.manifest.canvases
+        card_ids = list(cards.values_list('short_id', flat=True))
+        
+        # create a paginator with 1 card per page and get the current "page"
+        paginator = Paginator(card_ids, 1)
+        current_index = card_ids.index(self.object.short_id)
+        page_obj = paginator.page(current_index + 1) # 1-based page index
```

Comment:
Can we come up with a better name for `page_obj` ? Maybe something like `card_paginator_page` ? (not great, but  at least more obvious what it is)

Reply:
does `card_page` make sense?

Reply:
... I think that sounds ok. Better than `page_obj` !